utils/polus-imagej-util/{{cookiecutter.project_slug}}/plugin.json

```diff
@@ -20,19 +20,27 @@
       },
     {%- endif %}
       "description": "{{ val.description }}",
-      "required": {{ val.required.lower() }}
+      "required": {{ val.required.lower() }}{%- if val.type=='array' %},
+      "options":{
+        "items": {
+          "type": "string",
+          "title": "{{ val.description }}",
+          "widget": "string"
+        }
+      }
```

Comment:
Is seems like this is only applicable for array types, yet this looks like it will be applied to all inputs.

Reply:
I have tested this to make sure it is only applied to `array` types

Reply:
Also this file is no longer part of the PR as the changes to the pipeline have now been merged: [imagej-ui PR](https://github.com/PolusAI/polus-plugins/pull/229)